src/runtime_src/core/include/experimental/xrt_ip.h

```diff
@@ -11,8 +11,10 @@
 #include "xrt/detail/pimpl.h"
 
 #ifdef __cplusplus
+#include <bitset>
```

Comment:
Do the space.  It's common practice to add spaces corresponding to the current level between `#` and `whatever` (include).

Reply:
done